tests/modules/nf-core/angsd/contamination/main.nf

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ANGSD_CONTAMINATION } from '../../../../../modules/nf-core/angsd/contamination/main.nf'
+include { ANGSD_DOCOUNTS } from '../../../../../modules/nf-core/angsd/docounts/main.nf'
+
+workflow test_angsd_contamination {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_markduplicates_sorted_bam'], checkIfExists: true),
+        file(params.test_data['homo_sapiens']['illumina']['test_paired_end_markduplicates_sorted_bam_bai'], checkIfExists: true),
+        []
+    ]
+
+    hapmap_file = file("https://github.com/nf-core/eager/raw/master/assets/angsd_resources/HapMapChrX.gz")
```

Comment:
```suggestion
    hapmap_file = [ [id:'test2'], file("https://github.com/ANGSD/angsd/raw/0.940/RES/HapMapChrX.gz")]
```
Applied the suggestion to add a meta by @nvnieuwk, and additionally changed the URL for the file to the ANGSD github page from the eager one. :)

Reply:
Can these files be added to the modules branch in the test-datasets repo? Or is it a huge file?

Reply:
That would be the cleanest solution for sure. Will do that before merging this in for sure.

Reply:
Awesome!